cypress/integration/login.js

```diff
@@ -8,20 +8,28 @@ describe('Sourcebooks login', function() {
         cy.get('.Select.not-valid').should('be.visible')
     })
 
-    it('Should be able to login with role User', function () {
+    it('Should be able to login with role Team Lead', function () {
 
         cy.visit('/');
         cy.get('[id="loginForm.userId"]').click({force:true});
         cy.get('[aria-label="Demo User"]').click();
         cy.get('[id="loginForm.role"]').click({force:true});
-        cy.get('[aria-label="User"]').click();
+        cy.get('[aria-label="Team Lead"]').click();
         cy.get('[type="submit"]').click();
 
         cy.url().should('include', '/time-logging');
         cy.get('.page__title').contains('Timesheets')
         cy.get('.calendar').should('be.visible')
         cy.get('.tile.form').should('be.visible')
         cy.get('.user-info__title').contains('Demo User');
-        cy.get('.main-nav').find('li').should('have.length', 1);
+        cy.get('.main-nav').find('li').should('have.length', 2);
+    })
+
+    it('Should be displaying todays date in Time Logging page', function () {
+        const today = new Date();
+        const date = today.getDate();
+        const mont = today.getMonth();
+
+        cy.get('[class="calendar__day calendar--today calendar--selected"]').find('[class="calendar__date"]').contains(date);

```

Comment:
In your new pull request, I think just getting ".calendar--today" would be enough and it would reduce this long selector chain